create_project_workspace.py

```diff
@@ -68,11 +74,15 @@ def main(argv=None):
     parser = argparse.ArgumentParser(description="Bazel Federation WORKSPACE Generation Script")
     parser.add_argument("--project", type=str, required=True)
     parser.add_argument("--repo", type=str)
+    parser.add_argument("--has_internal_deps", type=bool)
```

Comment:
replace type=bool with action='store_true'

This allows us to pass '--has_internal_deps' without a value. The current code would require setting  '--has_internal_deps=true' in the yaml files, and that's why CI is failing right now.